default/python/AI/AIstate.py

```diff
@@ -400,6 +400,7 @@ def __update_system_status(self):
         supply_unobstructed_systems = set(empire.supplyUnobstructedSystems)
         min_hidden_attack = 4
         min_hidden_health = 8
+        observed_enemies = self.misc.setdefault("observed_enemies", set())
```

Comment:
When you introduce new persistent collection, you can add some documentation about it. 

- what object does it contain
- when object added to it
- when object removed from it